### PR TITLE
Make pylint happy

### PR DESCRIPTION
**Pull request by mimi1vx**
Disable some warnings

sort imports and deduplicate some code

### CODE REVIEW

File: openqa_bugfetcher/issues/bugzilla_gnome_issue.py [hunk 1]
```diff
@@ -1,35 +1,6 @@
...
-class BugzillaGnomeIssue(BaseIssue):
+class BugzillaGnomeIssue(BugzillaBaseIssue):
     prefixes = {"bgo"}
-
-    def fetch(self, conf):
-        def json_rpc_get(url, method, params):
-            get_params = OrderedDict({"method": method, "params": json.dumps([params])})
-            return requests.get(url, params=get_params)
-
-        url = "https://bugzilla.gnome.org/jsonrpc.cgi"
```
Comment (aylen384) [inline openqa_bugfetcher/issues/bugzilla_gnome_issue.py:17]: I think you forgot to set the url accordingly for each bugzilla subclass.

**Review by aylen384 — CHANGES_REQUESTED** [review]

File: openqa_bugfetcher/issues/bugzilla_gnome_issue.py (hunk 1 above)
Comment (mimi1vx) [reply openqa_bugfetcher/issues/bugzilla_gnome_issue.py:17]: oh, thx

**Comment by kalikiana** [comment]
I don't see pylint checks here 🤔 
image

**Review by aylen384 — COMMENTED** [review]
maybe limit them on the master branch and PRs to that branch:
https://github.com/os-autoinst/openqa_bugfetcher/pull/6

**Comment by mimi1vx** [comment]
> maybe limit them on the master branch and PRs to that branch:
> #6

ok, change in action removed, to use @asdil12  #6

**Comment by aylen384** [comment]
> ok, change in action removed, to use @asdil12 #6

I merged #6 so that you can rebase and run the tests in this PR.

**Review by aylen384 — APPROVED** [review]

**Pull request merged**